test/NuGet.Core.Tests/NuGet.Packaging.Test/ContentModelTests/ContentModelBuildTests.cs

```diff
@@ -4,14 +4,16 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 using NuGet.ContentModel;
 using NuGet.Frameworks;
 using NuGet.RuntimeModel;
 using Xunit;
 
 namespace NuGet.Client.Test
 {
+    /// <summary>
+    /// These unit test cases cover <see cref="ContentModel.Infrastructure.PatternExpression.Match(string, IReadOnlyDictionary{string, ContentPropertyDefinition})"/>
+    /// </summary>
```

Comment:
Two things::

1. My assumption is that you created your previous PR, that is now closed, because you looked at `PatternExpression.cs`, and couldn't find tests, so you spent time writing tests. While this comment makes it possible to use "find in all files" to look for the class name, I think it would be more discoverable if there was a comment in `PatternExpression.cs` and `PatternDefinition.cs`, saying that tests for the classes are located in `NuGet.Packaging.Test/ContentModelTests/`

2. The `///` comments are shown by Intellisense when you hover over types symbols, and some other scenarios. While it's not a "problem" here, because there's no reason for anyone to reference a test class from any other file, I don't think it's useful information to show in intellisense. So, if you add comments to `PatternDefinition.cs` and `PatternExpression.cs`, I recommend just using `//` (two slash) comments.

Reply:
These are two good points. @donnie-msft what do you think?

Reply:
1. Finding references is far superior to me than text based searching. If Andy is suggesting that it's important to have a comment in both the caller and receiver, then you can consider having both comments, but I see no reason to remove this one, regardless.

2. I think it's useful information to link a type being tested directly to the test, when otherwise, there's no way to know without following the call stack manually. If you use 2 slashes, you'll not be able to reference the type in the comment in a way that VS can detect it.